src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCar = 40;
```

Comment:
Consider using a constant for the daily rental cost to make the code more readable and maintainable. This aligns with the checklist item: 'Don't use literals in expressions (HARDCODE). Instead use constants explaining this values.'